README.md

```diff
@@ -1,5 +1,10 @@
 # Upgrade Assistant
 
+## Share your feedback on the .NET Upgrade Assistant!
+We're interested to hearing how your experience with the .NET Upgrade Assistant has been going as you upgrade your project(s) from .NET Framework to .NET 5.
```

Comment:
Since we can move to other version of .net core probably good idea to mention that.